Makefile

```diff
@@ -75,4 +78,14 @@ remove_untagged_images:
 # Docker TEST
 
 flake8_staging: build_staging
-	$(staging_compose_cmd) run --rm --no-deps staging_web flake8
\ No newline at end of file
+	$(staging_compose_cmd) run --rm --no-deps staging_web flake8
+
+test_staging_web: generate_maps_staging
+	$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh dados
+	$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh dbf
+	$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh gis
+	$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh api
+	#$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh forecast
```

Comment:
is there a reason for this? if there is a reason, could you open an issue and add a comment with the issue number?